.gitignore

```diff
@@ -52,6 +52,7 @@ products/
 installer_osx/installer
 installer_osx/Install_Surge_*.dmg
 build_logs/
+.DS_Store
```

Comment:
I don't think this relates to spdlog

Reply:
That is correct; it relates to clean up mac build, which should be a separate PR as I commented below.